.github/workflows/build_packages.yaml

```diff
@@ -375,9 +435,16 @@ jobs:
           - emqx-edge
           - emqx
           - emqx-enterprise
-        # NOTE: for docker, only support latest otp version, not a matrix
+          - emqx-elixir
```

Comment:
Reminder: change tag name to have a suffix `-elixir` and keep the image as `emqx`